.github/workflows/release.yml

```
@@ -0,0 +1,58 @@
+      - v*.*.*
+
+env:
+  HUSKY: '0'
+
+jobs:
+  build:
+    if: github.repository == 'release-drafter/release-drafter'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.10)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)